<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>206</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>645</width>
    <height>206</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>645</width>
    <height>206</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="RightImport">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>89</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ValidateVar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="RightPathPreview">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>460</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LeftImport">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>89</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Image</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LeftPathPreview">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>460</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelVar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
